routing/integrated_routing_test.go

```diff
@@ -0,0 +1,57 @@
+package routing
+
+import (
+	"testing"
+)
+
+// TestProbabilityExtrapolation tests that probabilities for tried channels are
+// extrapolated to untried channels. This is a way to improve pathfinding
+// success by steering away from bad nodes.
+func TestProbabilityExtrapolation(t *testing.T) {
```

Comment:
this is very nice :)

Would it be possible to have it in its own sub package? Would possibly need to refactor the internal findPath call to export it, but feels like `routing` is big enough as is.

Reply:
It is possible of course, but I don't see much gain from that. We'd make the file listing of `routing` two files shorter, but have to pay for that by exporting symbols that we only need for testing. It is too bad that golang doesn't allow access to internals from test packages.